### PR TITLE
Reporter and WebHooks doc tweaks

### PR DESCRIPTION
**Pull request by WayneKeenan**
Changes to docs:

- GitHubCommentPush max paylod size note.
- Example code clarifications.
- Reverse proxy note.

### CODE REVIEW

File: master/docs/manual/cfg-wwwhooks.rst [hunk 1]
```diff
@@ -89,7 +89,7 @@ GitHub hook
 .. note::
 
    There is a standalone HTTP server available for receiving GitHub notifications as well: :contrib-src:`master/contrib/github_buildbot.py`.
-   This script may be useful in cases where you cannot expose the WebStatus for public consumption.
+   This script may be useful in cases where you cannot expose the WebStatus for public consumption. Alternatively, you can setup a reverse proxy :ref:`Reverse_Proxy_Config`
```
Comment (tardyp) [inline master/docs/manual/cfg-wwwhooks.rst]: well the Reverse_Proxy_Config exposes the whole web status to public.

You need a different config than the one that is described there.

File: master/docs/manual/cfg-wwwhooks.rst [hunk 2]
```diff
@@ -135,7 +135,8 @@ The simplest way to use GitHub hook is as follows:
 
 .. code-block:: python
 
-    c['www'] = dict(...,
+    c['www'] = dict(    
+        'base': True
```
Comment (tardyp) [inline master/docs/manual/cfg-wwwhooks.rst]: why? no need to have base!

Comment (WayneKeenan) [reply master/docs/manual/cfg-wwwhooks.rst]: it didn't work without it.

File: master/docs/manual/cfg-wwwhooks.rst (hunk 1 above)
Comment (WayneKeenan) [reply master/docs/manual/cfg-wwwhooks.rst]: so, either it's a documented security warning? or an example config?

Comment (tardyp) [reply master/docs/manual/cfg-wwwhooks.rst]: I think you can add an example here of an nginx config which implements the redirection just for the hook.

File: master/docs/manual/cfg-wwwhooks.rst (hunk 2 above)
Comment (tardyp) [reply master/docs/manual/cfg-wwwhooks.rst]: what didn't work? What is the symptom?

c['www']['base'] is not an available config
c['www']['change_hook_dialects']['base'] is, but only enables  the base hook. 

The github hook should not be dependent on the base hook. I double checked on the code that there is no code smell in that direction. Please double check on your side that this is not a confusion.

Comment (WayneKeenan) [reply master/docs/manual/cfg-wwwhooks.rst]: my confusion, I reverted back to PyPi release and it's ok. I must have tested (what I thought was) the endpoint using http://cam1.local:8010/change_hook/

Comment (tardyp) [reply master/docs/manual/cfg-wwwhooks.rst]: can you fix your patch then?

**Pull request merged**